### PR TITLE
update the context length header

### PR DESCRIPTION
**Pull request by quentin-**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -51,6 +51,10 @@ export default class RestAdapter {
 
         const body = response.body instanceof Object ? JSON.stringify(response.body) : response.body;
 
+        if (res.getHeader('Content-Length') != null) {
```
Comment (rgoldfinger) [inline src/index.js]: why is it not:
- `res.getHeader('Content-Length') == null` (or `!res.getHeader('Content-Length')`) ?
- `!=` rather than `!==`?

Comment (quentin-) [reply src/index.js]: `!res.getHeader('Content-Length')` is true for 0
`res.getHeader('Content-Length') == null` is the opposite of what we want
`res.getHeader('Content-Length') !== null` is false for undefined

Comment (rgoldfinger) [reply src/index.js]: I guess, why do you want to set it only when it's already set? Why is it wrong?

Comment (quentin-) [reply src/index.js]: I want to correct the header when it's wrong, if consumer don't want to send a content-length header for some reason than is should remain unset.

Comment (rgoldfinger) [reply src/index.js]: Any idea why it’s wrong?

**Pull request merged**